deploy/examples/cluster.yaml

```diff
@@ -166,6 +166,9 @@ spec:
 #    osd:
 #    cleanup:
 #    prepareosd:
+# clusterMetadata annotations will be applied to only `rook-ceph-mon-endpoints` configmap and the `rook-ceph-mon` and `rook-ceph-admin-keyring` secrets.
+# And clusterMetadata annotations will **not** be merged with `all` annotations.
+#    clusterMetadata:
```

Comment:
Can we actually get the example annotations here for using the `kubed` tool?

Reply:
added